File: app/src/main/java/com/rinconadadelsur/sistemas/bioseguridad/Entidades/eRegEliminar.java
package com.rinconadadelsur.sistemas.bioseguridad.Entidades;

public class eRegEliminar {
    String _id;
    String _descripcion;

    public eRegEliminar(String _id, String _descripcion) {
        this._id = _id;
        this._descripcion = _descripcion;
    }
    public eRegEliminar() {
    }

    public String get_id() {
        return _id;
    }

    public void set_id(String _id) {
        this._id = _id;
    }

    public String get_descripcion() {
        return _descripcion;
    }

    public void set_descripcion(String _descripcion) {
        this._descripcion = _descripcion;
    }
}

File: app/src/main/java/com/rinconadadelsur/sistemas/bioseguridad/Entidades/eRegEliminar.kt
package com.rinconadadelsur.sistemas.bioseguridad.Entidades

class eRegEliminar {
    var _id: String? = null
    var _descripcion: String? = null

    constructor(_id: String?, _descripcion: String?) {
        this._id = _id
        this._descripcion = _descripcion
    }

    constructor()

    fun get_id(): String? {
        return _id
    }

    fun set_id(_id: String?) {
        this._id = _id
    }

    fun get_descripcion(): String? {
        return _descripcion
    }

    fun set_descripcion(_descripcion: String?) {
        this._descripcion = _descripcion
    }
}
